spec/controllers/rooms_controller_spec.rb

```diff
@@ -14,7 +14,9 @@
 
   describe '#index' do
     it 'ids of rooms in response are matching room ids that belong to current_user' do
-      rooms = create_list(:room, 5, user:)
+      shared_rooms = create_list(:room, 2)
+      user.shared_rooms << shared_rooms
+      rooms = create_list(:room, 3, user:) + shared_rooms
```

Comment:
Minor improvement to test efficiency by handling the case for when having a list of owned and shared with rooms,